zerver/lib/user_mutes.py

```diff
@@ -29,9 +29,11 @@ def add_user_mute(
     )
 
 
-def remove_user_mute(user_profile: UserProfile, muted_user: UserProfile) -> None:
-    MutedUser.objects.get(user_profile=user_profile, muted_user=muted_user).delete()
+def get_mute_object(user_profile: UserProfile, muted_user: UserProfile) -> Optional[MutedUser]:
+    mute_object = None
+    try:
+        mute_object = MutedUser.objects.get(user_profile=user_profile, muted_user=muted_user)
+    except MutedUser.DoesNotExist:
+        pass
 
-
-def user_is_muted(user_profile: UserProfile, muted_user: UserProfile) -> bool:
-    return MutedUser.objects.filter(user_profile=user_profile, muted_user=muted_user).exists()
+    return mute_object
```

Comment:
This would be cleaner with a return in both code paths.

Reply:
Yes, definitely cleaner! Made this change.